OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -663,7 +663,7 @@ public void trackPLSCourseDatesShift(@NonNull String courseId, @NonNull String e
     public void trackValuePropModalView(@NonNull String courseId, @Nullable String assignmentId, @NonNull String screenName) {
         final FirebaseEvent event = new FirebaseEvent(Events.VALUE_PROP_MODAL_VIEW);
         event.putCourseId(courseId);
-        if (TextUtils.isEmpty(assignmentId)) {
+        if (!TextUtils.isEmpty(assignmentId)) {
```

Comment:
haha good catch (fix) .. would've been awkward seeing nulls for all assignmentsIds 😆